stdlib/LinearAlgebra/src/matmul.jl

```diff
@@ -694,7 +694,7 @@ parameter `tM` as follows:
 | --- | :--- | :--- |
 | `'N'` | `B[ir_dest, jr_dest]` | `M[ir_src, jr_src]` |
 | `'T'` | `B[ir_dest, jr_dest]` | `transpose(M)[ir_src, jr_src]` |
-| `'C'` | `B[ir_dest, jr_dest]` | `adjoint(M)[ir_src, jr_src]` |
+| `'C'` | `B[ir_dest, jr_dest]` | `conj(transpose((M))[ir_src, jr_src]` |
```

Comment:
Is there a case where this would differ from `adjoint(M)`?

Reply:
Yes, ` adjoint` acts recursively, whereas `conj(transpose(M))` can handle only a single layer. The issue is the implementation, not the docs here. The implementation copies the transpose, and then calls `conj!` on the elements of `B[ir_dest, jr_dest]`. But there's no `conj!` method for matrices of matrices. Even if we added such a method, that would likely performant worse than a direct `copy_adjoint!`.

Reply:
> Yes,  adjoint acts recursively, whereas conj(transpose(M)) can handle only a single layer.

But `transpose` and `conj` are also recursive?

Reply:
I hope/think I fixed it. We could additionally have a `conj!` method acting on arrays of arrays, but we could add that in another PR.